packages/nuxt3/src/app/composables/cookie.ts

```
@@ -65,7 +65,7 @@ function writeClientCookie (name: string, value: any, opts: CookieSerializeOptio
 }
 
 function writeServerCookie (event: CompatibilityEvent, name: string, value: any, opts: CookieSerializeOptions = {}) {
-  if (event) {
+  if (event && !event.res.writableEnded) {
```

This is not fixing any issue but silently ignoring a cookie addition that is expected but is not possible anymore!

---

it's an internal function so I think we could easily rename. I would not expect this to error; I think a warning would be okay.

---

It is used by `useCookie`. Silent errors are not a fix for lifecycle issues!